### PR TITLE
Update btf example

### PR DESCRIPTION
**Pull request by chenhengqi**

### CODE REVIEW

File: BTF-playground/btf_module_read.c [hunk 1]
```diff
@@ -66,17 +65,24 @@ int main(int argc, char **argv)
...
+		pr_err("ERROR(%d): btf__load_module_btf() module_name: %s\n",
 		       err, module_name);
 		goto out;
 	}
 
-        type_id = btf__find_by_name(module_btf, symbol_name);
+	type_id = btf__find_by_name(module_btf, symbol_name);
+	if (type_id < 0) {
+		pr_err("ERROR(%d): btf__find_by_name() symbol_name: %s\n",
+		       err, symbol_name);
```
Comment (tohojo) [inline BTF-playground/btf_module_read.c:76]: The error is not actually in `err` at this point, so this print is wrong; but it probably should just be propagated to err, so the program exit code is set correctly...

**Review by tohojo — CHANGES_REQUESTED** [review]
Thank you for the fixes! Just one small nit, otherwise LGTM :)

File: BTF-playground/btf_module_read.c (hunk 1 above)
Comment (chenhengqi) [reply BTF-playground/btf_module_read.c:77]: Updated.

**Comment by netoptimizer** [comment]
Thanks for improving this example.
LGTM

**Review by tohojo — APPROVED** [review]

**Pull request merged**